src/sentry/tasks/post_process.py

```diff
@@ -44,6 +44,43 @@ def _get_service_hooks(project_id):
     return result
 
 
+def _should_send_error_created_hooks(project):
+    from sentry.models import ServiceHook, Organization
+    from sentry import options
+    import random
+
+    cache_key = u'servicehooks-error-created:1:{}'.format(project.id)
+    result = cache.get(cache_key)
+
+    if result is None:
```

Comment:
If you plan to use the sampling rate to keep the load under control (and possibly constant), then you may want to apply the sampling rate on the cached value, instead of filling the cache with a sampled value. 
Otherwise you will have a spiky load with 100% of errors for a given project being sent to the webhook for 60 seconds periods when the cache says True no matter what the sampling rate is.

Reply:
> If you plan to use the sampling rate to keep the load under control (and possibly constant)

Yes, this is for us to manually test the load and slowing increase the %, so yeah makes sense to not cache this value